.yarn/versions/b87032c4.yml

```diff
@@ -0,0 +1,27 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/plugin-pnp": patch
+  "@yarnpkg/pnp": patch
```

Comment:
Is there a risk in shipping it as a patch? Entirely changing the implementation may cause a couple of behaviours that people rely on to change.

Reply:
Yes, this is a breaking change but other PRs making PnP match the Node.js behaviour usually aren't marked as such even though all of them are breaking changes.

Our current implementation doesn't match any version of Node.js while this new version doesn't match versions before v18, the former seems worse.

https://xkcd.com/1172 comes to mind.